core/arch/arm/plat-rd1ae/platform_config.h

```diff
@@ -0,0 +1,40 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024, Arm Limited
+ */
+
+#ifndef PLATFORM_CONFIG_H
+#define PLATFORM_CONFIG_H
+
+#include <mm/generic_ram_layout.h>
+
+/* Make stacks aligned to data cache line length */
+#define STACK_ALIGNMENT			64
+
+#define DRAM0_BASE			0x80000000UL
+#define DRAM0_SIZE			0x80000000UL
+
+#define DRAM1_BASE			0x8080000000ULL
+#define DRAM1_SIZE			0x80000000ULL
```

Comment:
nitpicking: would prefer `UL(xxx)`/`ULL(xxx)` instead of `xxxUL`/`xxxULL`

Reply:
Done